services/graph/pkg/identity/ldap.go

```diff
@@ -60,6 +73,21 @@ type userAttributeMap struct {
 
 type ldapAttributeValues map[string][]string
 
+// ParseDisableMechanismType checks that the configuration option for how to disable users is correct.
+func ParseDisableMechanismType(disableMechanism string) (DisableUserMechanismType, error) {
+	disableMechanism = strings.ToLower(disableMechanism)
+	switch disableMechanism {
+	case "", "none":
```

Comment:
Nit, but this would probably be cleaner if using a map, with a default of disable if the key not found. Would also make it easier to extend later.